app/models/identity_verification.rb

```diff
@@ -0,0 +1,5 @@
+class IdentityVerification < Verification
+  def verification_type

```

Comment:
A thought- are these methods needed?  Maybe they should be defined by the bottom-most subclass, e.g. `AlloyIdentityVerification`, but not here?
Maybe this method should be like
```
def verification_type
  fail NotImplementedError
end
```
?

Reply:
They are called in the view somewhere - which is why there's a hack comment re:I18N

Reply:
@MatthewSchultz Yup, I understand that they're called, but I'm suggesting that they could be implemented in the subclass, rather than in the superclass.  Adding `fail NotImplementedError` to the superclass forces developers to implement in the subclass.